### DISCUSSION TITLE
Sales invoice export to csv

### DISCUSSION THREAD

**Discussion by Gazzxy** [YetiForceCompany/YetiForceCRM#15186] [How to]
Hi was hoping we could use the export to csv function to export all invoices to sage. but now we have a few real and big invoices in the system the export system seems completely useless for anything to be brutally

is there a way to filter this down to useful information straight from the export

**Comment by mariuszkrzaczkowski** [comment]
try this
![image](https://user-images.githubusercontent.com/10198654/133556343-e00fabf8-a2c4-48f5-9235-0e09541ce5e9.png)
![image](https://user-images.githubusercontent.com/10198654/133556377-81b09afb-c15d-4b17-b67f-08de39966feb.png)
![image](https://user-images.githubusercontent.com/10198654/133556391-4dcf1de1-690b-46b7-8ce5-be4f83d6dd09.png)

**Comment by Gazzxy** [comment]
this still gives us a csv with a lot of information and just winds up looking like there's duplicates when it has an entry for each item on an invoice..
just need it to export a report of the invoices in the system and the essential info (invoice number, net total, date ect)

**Reply by rskrzypczak** [reply]
From version 6.3, which has not yet been released, it will be possible.
![image](https://user-images.githubusercontent.com/10221999/136221213-f1176c49-e4dc-4bcf-aabc-a6fa01de12e6.png)

**Reply by Gazzxy** [reply]
that would work much better

**Comment by noelhunter** [comment]
1. Create a filter on invoices that displays only the fields you want
2. Select all of the invoices
3. Choose "Export to Excel"

This works for me, containing just the fields displayed.  However, export to CSV seems to still contain all fields, even if I select the option to export just what is on the page.

**Comment by Gazzxy** [comment]
Actually export ot excel is clean not working for me at the min.. just makes corrupt file that cannot be opened intelligibly
